### ISSUE TITLE
Removal and deprecation of broken versions of email-cleaner

### ISSUE THREAD

**Issue by destroyer22719** [destroyer22719/email-cleaner#5]
# Why several email-cleaner versions are being deprecated and removed

TL;DR - A lot of versions had issues so I deprecated or deleted the versions leaving only `1.3.1` and `1.1.1` working

Thank you for using email-cleaner, as my first npm package I have learned and explored a lot. That inevitably means me running into errors which leads to some packages having issues or just straight up not working at all. Because of that I am going to mend my mistakes and remove and deprecate several broken versions of email-cleaner.

# Versions Affected
- 0.9.0-0.9.2 [removed]
- 0.9.3-1.1.0 [deprecated]
- 1.1.2-1.3.0 [removed]

## 0.9.0-0.9.2 - removed
These versions I was experimenting with has issues with importing. Due to how typescript compiles the code thus will be removed.

## 0.9.3-1.1.0 - deprecated
This version has issues with the configuration option of `overrideDefaultCases` not working at all 

## 1.1.2-1.3.0 - removed
These version has issues with importing with both commonjs and es6 modules due to the typescript not being compiled properly.
